File: kyu_7/002_Disemvowel_Trolls/javascript/js/main.js
// solution 1
function disemvowel(str) {
  let new_arr = str.split("");

  let str_fill = new_arr.filter(ele => !"aeiou".includes(ele.toLowerCase()));

  return str_fill.join("");
}

console.log(disemvowel("This website is for losers LOL!"));

// solution 2
function disemvowel(str) {
  return str.replace(/[aeiou]/gi, "");
}

console.log(disemvowel("This website is for losers LOL!"));

File: kyu_7/002_Disemvowel_Trolls/python/main.py
# solution 1
def disemvowel(string_):
    new_str = []

    for char in string_:
        if char.lower() not in "aeiou":
            new_str.append(char)

    return "".join(new_str)

print(disemvowel("This website is for losers LOL!")) # Ths wbst s fr lsrs LL!

